engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -3070,7 +3071,7 @@ public void execute(String[] args, ConsoleReader reader) throws Exception
             }
 
             arr.put(oper);
-          } catch (Exception | NoClassDefFoundError ex) {
+          } catch (Exception | NoClassDefFoundError | UnsupportedClassVersionError ex) {
```

Comment:
get-app-package-operators will throw this exception and kill the apexCli. Catching it there will still output correctly compiled operators.